### PR TITLE
Jinan/expose props to bundle

### PR DESCRIPTION
**Pull request by PorterNan**
# What

- Expose props to embed html
- Update composite bundle doc
- Add release github bundle file urls

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx [hunk 1]
```diff
@@ -7,8 +7,58 @@ import { Canvas, Title } from '@storybook/addon-docs';
 
 If you are not using React, you can still embed a calling/chat composite into your html page with a simple `<script>` tag and several lines of code like:
 
+You can easily find prebuilt bundles from:
```
Comment (prprabhu-ms) [inline packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:10]: Are we not following SemVer in releasing these bundles? Or is `releases/latest

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx [hunk 2]
```diff
@@ -7,8 +7,58 @@ import { Canvas, Title } from '@storybook/addon-docs';
...
+    await chatComposite.loadChatComposite( // This function will be exposed to window.chatComposite once you import composite bundle
+      {
+        displayName, // string
+        threadId, // string
+        endpointUrl, // string
+        userId, // Object { communicationUserId: string }
+        token // string
+      },
+      elementToRender, // Container element
+      props // Object for all props(except adapter) of composite, optional, check composite page for more details
```
Comment (prprabhu-ms) [inline packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:33]: I couldn't understand what 

> Object fo all props(except adapter)

means.

**Review by prprabhu-ms — APPROVED** [review]

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx (hunk 1 above)
Comment (PorterNan) [reply packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:10]: Yep, this is alwayls pointing you to the latest beta release

**Review by alcail — APPROVED** [review]

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx (hunk 2 above)
Comment (PorterNan) [reply packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:33]: Updated, this is an object contains all optional props for the adapter, like showTitle, showParticipant

**Pull request merged**